### DISCUSSION TITLE
Custom Model in Monai Bundle

### DISCUSSION THREAD

**Discussion by vikashg** [Project-MONAI/tutorials#853] [Q&A]
Hello everyone, 
So I have been going through the MONAI BUNDLE module and looking at several bundles that are already shared. I had a question regarding building a model using a custom model. 
Most of the models that are shared on the model zoo are based on the model definitions in `monai.networks.nets`, which also contain the keywords `blocks_down` and `blocks_up` as in `SegResNet`. These keywords need to be specified in the `train.json`.

I was wondering if I have a custom model defined in a file for example `model_def.py`. How can I build a model based on this custom model. In addition is there a method to load and re-train a model. I tried to look into the [custom_component example](https://github.com/Project-MONAI/tutorials/blob/main/modules/bundle/custom_component/configs/custom_train.json). But it was not very clear.

**Comment by Nic-Ma** [comment]
Hi @vikashg ,

You can refer to this PR: https://github.com/Project-MONAI/model-zoo/pull/70

Thanks.
